### ISSUE TITLE
More info for unicornd troubleshooting? 

### ISSUE THREAD

**Issue by ferventgeek** [pimoroni/unicorn-hat#126]
Trying to start unicornd from a service, but it's dying with only the limited debug output from systemd at the bottom. Is there a better way to troubleshoot this? Pi 3 A+. Thx. 

unocornd.service
```
[Unit]
Description=Unicorn HAT daemon

[Service]
ExecStart=/usr/sbin/unicornd

[Install]
WantedBy=multi-user.target
```

journalctl -u unicornd.service
```
Dec 22 21:12:47 cuepi systemd[1]: Started Unicorn HAT daemon.
Dec 22 21:12:47 cuepi systemd[1]: unicornd.service: Main process exited, code=exited, status=1/FAILURE
Dec 22 21:12:47 cuepi systemd[1]: unicornd.service: Unit entered failed state.
Dec 22 21:12:47 cuepi systemd[1]: unicornd.service: Failed with result 'exit-code'.
```

**Comment by ferventgeek** [comment]
@Gadgetoid This is the issue the tweet was about. Any advice would be deeply appreciated. :)

**Comment by Gadgetoid** [comment]
Must be related to https://github.com/pimoroni/unicorn-hat/issues/131 for which I've pushed an, admittedly not very timely, fix.

The trouble is that `unicornd` uses a git submodule to include the `rpi_ws281x` library, so it doesn't get automatically updated as new versions of the Pi are released. I've bumped it to the latest version, which should include support up to the Pi 4.

Better debugging would require `unicornd` to report the error code it gets from `rpi_ws281x`. Possible, I believe, but needs some time and tlc to pull off.
